.env.local

```diff
@@ -1,4 +1,9 @@
+# Database 
 DB_CONN_STRING="mongodb+srv://admin:admin64@gamingbacklog.ce5ec.mongodb.net/libraries?retryWrites=true&w=majority"
 DB_NAME="gamebacklogDB"
 LIBRARY_COLLECTION="libraries"
 GAME_INSTANCE_COLLECTION="games"
+
+# IGDB 
+CLIENT_ID="gcicsxic8po9xjyr4x2105gxm0861s"
+CLIENT_SECRET="bnsej7verrbr55dgkdxqk8aocxl2o4"
```

Comment:
@julian3ng did the linter include newlines at the ends of files or do we still need to add that?

Reply:
created https://gaming-backlog.atlassian.net/browse/GB-30